sonar-helm-for-iac/src/converters/io-handling.go

```diff
@@ -31,6 +31,10 @@ import (
 
 var END_TOKEN = []byte("END\n")
 
+// The bigger input buffer prevent the flacky error: "error reading content: strconv.Atoi", see SONARIAC-1392
+// The default is 64 * 1024
+var INPUT_BUFFER_SIZE = 1024 * 1024 * 1024
```

Comment:
It is too hard to implement a reliable test for that. A part of `ibm-charts` repository where we see the problem is flaky and the exact reason is unclear.